packages/vscode-extension/src/utilities/subprocess.ts

```diff
@@ -16,8 +18,8 @@ async function getPathEnv(appRoot: string) {
 
   // Fish, bash, and zsh all support -i and -c flags.
   const shellPath = process.env.SHELL ?? "/bin/zsh";
-  const { stdout: path } = await execa(shellPath, ["-i", "-c", `cd "${appRoot}" && echo "$PATH"`]);
-  return path.trim();
+  const { stdout: path } = await execa(shellPath, ["-i", "-c", `cd "${appRoot}" && echo "${PRE_PATH_TOKEN}$PATH"`]);
```

Comment:
I'd suggest that to make it easier to debug such problems in the future we should add log statement to print the output of this command